data/mods/More_Survival_Tools/items.json

```diff
@@ -0,0 +1,571 @@
+[
```

Comment:
Most items are in this one file, I should've taken the time to separate more than just the armor.

Also, embercarrier/tinderbox should be working now, need some compile-happy players to test this.

Production of curds for hard cheese is still extra derpy, I should've looked to see if mucking about in the source could've fixed that like it hopefully fixed ember carriers.